components/Button/ScrollDown/index.js

```diff
@@ -0,0 +1,14 @@
+import { Button } from "@nextui-org/react"
```

Comment:
Comment out or remove completely this piece of code since it is not being used now (maybe store code for later need)